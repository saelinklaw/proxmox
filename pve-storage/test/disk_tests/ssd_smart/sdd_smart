smartctl 6.4 2014-10-07 r4002 [x86_64-linux-4.4.19-1-pve] (local build)
Copyright (C) 2002-14, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED

SMART Attributes Data Structure revision number: 4
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAGS    VALUE WORST THRESH FAIL RAW_VALUE
  5 Reallocated_Sector_Ct   -O--CK   100   100   ---    -    0
  9 Power_On_Hours          -O--CK   100   100   ---    -    799
 12 Power_Cycle_Count       -O--CK   100   100   ---    -    92
165 Unknown_Attribute       -O--CK   100   100   ---    -    9699447
166 Unknown_Attribute       -O--CK   100   100   ---    -    1
167 Unknown_Attribute       -O--CK   100   100   ---    -    46
168 Unknown_Attribute       -O--CK   100   100   ---    -    5
169 Unknown_Attribute       -O--CK   100   100   ---    -    1079
170 Unknown_Attribute       -O--CK   100   100   ---    -    0
171 Unknown_Attribute       -O--CK   100   100   ---    -    0
172 Unknown_Attribute       -O--CK   100   100   ---    -    0
173 Unknown_Attribute       -O--CK   100   100   ---    -    1
174 Unknown_Attribute       -O--CK   100   100   ---    -    22
184 End-to-End_Error        -O--CK   100   100   ---    -    0
187 Reported_Uncorrect      -O--CK   100   100   ---    -    0
188 Command_Timeout         -O--CK   100   100   ---    -    0
194 Temperature_Celsius     -O---K   073   064   ---    -    27 (Min/Max 23/64)
199 UDMA_CRC_Error_Count    -O--CK   100   100   ---    -    0
230 Unknown_SSD_Attribute   -O--CK   100   100   ---    -    146029805602
232 Available_Reservd_Space PO--CK   100   100   004    -    100
233 Media_Wearout_Indicator -O--CK   100   100   ---    -    1574
234 Unknown_Attribute       -O--CK   100   100   ---    -    2303
241 Total_LBAs_Written      ----CK   253   253   ---    -    2111
242 Total_LBAs_Read         ----CK   253   253   ---    -    1542
244 Unknown_Attribute       -O--CK   000   100   ---    -    0
                            ||||||_ K auto-keep
                            |||||__ C event count
                            ||||___ R error rate
                            |||____ S speed/performance
                            ||_____ O updated online
                            |______ P prefailure warning
