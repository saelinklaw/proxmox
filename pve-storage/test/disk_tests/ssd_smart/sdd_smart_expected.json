{
    "attributes" : [
	{
	    "id" : "  5",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "worst" : 100,
	    "raw" : "0",
	    "threshold" : 0,
	    "value" : 100,
	    "name" : "Reallocated_Sector_Ct"
	},
	{
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "  9",
	    "name" : "Power_On_Hours",
	    "threshold" : 0,
	    "value" : 100,
	    "raw" : "799"
	},
	{
	    "name" : "Power_Cycle_Count",
	    "raw" : "92",
	    "threshold" : 0,
	    "value" : 100,
	    "fail" : "-",
	    "worst" : 100,
	    "id" : " 12",
	    "flags" : "-O--CK"
	},
	{
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "165",
	    "name" : "Unknown_Attribute",
	    "threshold" : 0,
	    "value" : 100,
	    "raw" : "9699447"
	},
	{
	    "value" : 100,
	    "threshold" : 0,
	    "raw" : "1",
	    "name" : "Unknown_Attribute",
	    "flags" : "-O--CK",
	    "id" : "166",
	    "worst" : 100,
	    "fail" : "-"
	},
	{
	    "id" : "167",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "worst" : 100,
	    "raw" : "46",
	    "value" : 100,
	    "threshold" : 0,
	    "name" : "Unknown_Attribute"
	},
	{
	    "name" : "Unknown_Attribute",
	    "raw" : "5",
	    "value" : 100,
	    "threshold" : 0,
	    "fail" : "-",
	    "worst" : 100,
	    "id" : "168",
	    "flags" : "-O--CK"
	},
	{
	    "flags" : "-O--CK",
	    "id" : "169",
	    "worst" : 100,
	    "fail" : "-",
	    "value" : 100,
	    "threshold" : 0,
	    "raw" : "1079",
	    "name" : "Unknown_Attribute"
	},
	{
	    "raw" : "0",
	    "threshold" : 0,
	    "value" : 100,
	    "name" : "Unknown_Attribute",
	    "id" : "170",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "worst" : 100
	},
	{
	    "fail" : "-",
	    "worst" : 100,
	    "id" : "171",
	    "flags" : "-O--CK",
	    "name" : "Unknown_Attribute",
	    "raw" : "0",
	    "value" : 100,
	    "threshold" : 0
	},
	{
	    "name" : "Unknown_Attribute",
	    "raw" : "0",
	    "value" : 100,
	    "threshold" : 0,
	    "fail" : "-",
	    "worst" : 100,
	    "id" : "172",
	    "flags" : "-O--CK"
	},
	{
	    "name" : "Unknown_Attribute",
	    "threshold" : 0,
	    "value" : 100,
	    "raw" : "1",
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "173"
	},
	{
	    "name" : "Unknown_Attribute",
	    "value" : 100,
	    "threshold" : 0,
	    "raw" : "22",
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "174"
	},
	{
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "184",
	    "name" : "End-to-End_Error",
	    "value" : 100,
	    "threshold" : 0,
	    "raw" : "0"
	},
	{
	    "name" : "Reported_Uncorrect",
	    "value" : 100,
	    "threshold" : 0,
	    "raw" : "0",
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "187"
	},
	{
	    "name" : "Command_Timeout",
	    "raw" : "0",
	    "value" : 100,
	    "threshold" : 0,
	    "fail" : "-",
	    "worst" : 100,
	    "id" : "188",
	    "flags" : "-O--CK"
	},
	{
	    "threshold" : 0,
	    "value" : 73,
	    "raw" : "27 (Min/Max 23/64)",
	    "name" : "Temperature_Celsius",
	    "flags" : "-O---K",
	    "id" : "194",
	    "worst" : 64,
	    "fail" : "-"
	},
	{
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "199",
	    "name" : "UDMA_CRC_Error_Count",
	    "value" : 100,
	    "threshold" : 0,
	    "raw" : "0"
	},
	{
	    "name" : "Unknown_SSD_Attribute",
	    "threshold" : 0,
	    "value" : 100,
	    "raw" : "146029805602",
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "230"
	},
	{
	    "raw" : "100",
	    "value" : 100,
	    "threshold" : 4,
	    "name" : "Available_Reservd_Space",
	    "id" : "232",
	    "flags" : "PO--CK",
	    "fail" : "-",
	    "worst" : 100
	},
	{
	    "threshold" : 0,
	    "value" : 100,
	    "raw" : "1574",
	    "name" : "Media_Wearout_Indicator",
	    "flags" : "-O--CK",
	    "id" : "233",
	    "worst" : 100,
	    "fail" : "-"
	},
	{
	    "id" : "234",
	    "flags" : "-O--CK",
	    "fail" : "-",
	    "worst" : 100,
	    "raw" : "2303",
	    "value" : 100,
	    "threshold" : 0,
	    "name" : "Unknown_Attribute"
	},
	{
	    "fail" : "-",
	    "worst" : 253,
	    "id" : "241",
	    "flags" : "----CK",
	    "name" : "Total_LBAs_Written",
	    "raw" : "2111",
	    "value" : 253,
	    "threshold" : 0
	},
	{
	    "worst" : 253,
	    "fail" : "-",
	    "flags" : "----CK",
	    "id" : "242",
	    "name" : "Total_LBAs_Read",
	    "threshold" : 0,
	    "value" : 253,
	    "raw" : "1542"
	},
	{
	    "name" : "Unknown_Attribute",
	    "value" : 0,
	    "threshold" : 0,
	    "raw" : "0",
	    "worst" : 100,
	    "fail" : "-",
	    "flags" : "-O--CK",
	    "id" : "244"
	}
    ],
    "type" : "ata",
    "health" : "PASSED"
}
